packages/platform/src/editor/plugins/toolbar/BlockFormatDropDown.tsx

```diff
@@ -71,6 +73,9 @@ export default function BlockFormatDropDown({
         $setBlocksType(selection, () => $createParaNode(typeToStyle(selectedBlockType)));
       }
     });
+    if (closeMenu) {
+      closeMenu();
+    }
```

Comment:
FYI there is a shortcut syntax you can use here `closeMenu.?();` which says call this method if it is defined. That way it's only 1 line instead of 3. Nothing wrong with using the `if` block as it is but I think this other syntax is a little cleaner.